### ISSUE TITLE
White channel of RGBW lights no longer accessible via GUI 

### ISSUE THREAD

**Issue by sparkydave1981** [home-assistant/frontend#15990]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

Since the HA core 'dialog' update (2023.3) the white channel controls of my lights are no longer shown in the GUI.

I have a number of RGBW lights running ESPhome and previously the more-options dialog allowed for the white channel brightness to be adjusted separate to the colours. Now only the colour pallete comes up and there are no controls for the white channel at all.

### Describe the behavior you expected

Selectors to adjust both the colour brightness and the white brightness separately.

### Steps to reproduce the issue

1.turn on a RGBW light (mine are flashed with ESPhome)
2. go to 'more info' dialog
3. notice that there is no way to control the white channel brightness
...

### What version of Home Assistant Core has the issue?

2023.3.x

### What was the last working version of Home Assistant Core?

2023.2.5

### In which browser are you experiencing the issue with?

Google Chrome Version 111.0.5563.147

### Which operating system are you using to run this browser?

Windows 10

### State of relevant entities

```txt
RGBW light when ON
```

### Problem-relevant frontend configuration

```yaml
double_tap_action:
  action: more-info
entity: light.living_room_downlights
hold_action:
  action: ''
type: light
```

**Comment by piitaya** [comment]
Hello 👋 It's fixed by https://github.com/home-assistant/frontend/pull/15774 and will be available in `2023.4`.

**Comment by sparkydave1981** [comment]
> Hello 👋 Are you talking about white mode ?
> This PR #15774 will be available in `2023.4`.

yes, that's it.  glad to see it will be fixed soon.  thanks

**Comment by sparkydave1981** [comment]
I just updated the ESPhome version on a couple of these affected lights and the white channel and colours are now both separately brightness adjustable again via the UI.  It would seem that the ESPhome version update has also fixed the issue that was caused by HA 2023.3.x.  Strange.
